### ISSUE TITLE
Change "twist" from a string to an object

### ISSUE THREAD

**Issue by johnjenkman** [samjbmason/punkapi#43]
The twist property is currently a string. However, there are numerous recipes that have multiple twist/brewhouse additions with various properties.

The twist property should be an array object similar to the hops property to support multiple additions.

In DIY Dog 2018 you can look at AB:24 as an example.

As a workaround for now they could be added under the hops as many of these additions are added during the dry hop stage or to the kettle during flame out or whirlpool.

**Comment by sammdec** [comment]
Yeah this makes a lot of sense, when I open the v3 branch we can put it in there
